swiper/swiper.d.ts

```diff
@@ -53,6 +53,8 @@ interface SwiperOptions {
     // Navigation
     keyboardControl?: boolean;
     mousewheelControl?: boolean;
+    nextButton?: string;
```

Comment:
@mhegazy Unsure. But I guess it could be since we're adding classname as parameter, Swiper then finds the html element by classname.